src/twisted/conch/ssh/filetransfer.py

```diff
@@ -475,7 +478,7 @@ def _ebStatus(self, reason, requestId, msg=b"request failed"):
             elif reason.value.errno == errno.EEXIST:
                 code = FX_FILE_ALREADY_EXISTS
             else:
-                log.err(reason)
+                self.log.failure(nativeString(message), reason)
```

Comment:
Since `message` is caller-controlled it might not be a valid format string. I don't think that it can be used directly. Perhaps we can also add a bit of context to the message?

```suggestion
                self.log.failure("Request {requestId} failed: {message}", reason, requestId=requestId, message=message)
```